src/backend/metal/src/device.rs

```diff
@@ -1643,7 +1646,25 @@ impl hal::Device<Backend> for Device {
             },
         };
 
-        Ok(n::ImageView(image.raw.new_texture_view(mtl_format)))
+        if swizzle != format::Swizzle::NO {
```

Comment:
Do we have any ideas about how to handle this already?

Reply:
Actually, no. I think we'll have this as a flag for VkPI: swizzling is not allowed (on Metal)